src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Collect.txt

```diff
@@ -42,13 +42,13 @@ Errors: Error 12-15: Invalid argument type (Text). Expecting a Record value inst
 Errors: Error 12-13: Invalid argument type (Number). Expecting a Record value instead.|Error 12-13: Invalid argument type. Cannot use Number values in this context.|Error 0-14: The function 'Collect' has some invalid arguments.
 
 >> Collect(Table({name: "VC"}), {surname: "textInput1"})
```

Comment:
>Collect [](http://example.com/codeflow?start=3&length=7)

New test:
can we confirm that Collect works with a subset?

Collect(Table({x:1,y2}), {x: 3}).y 
Blank()

Reply:
Passed.